### DISCUSSION TITLE
Welcome to Testomania Discussions!

### DISCUSSION THREAD

**Discussion by Nodrex** [Nodrex/Testomania#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Nodrex** [comment]
Since private free repo does not give much of control over branches and pull requests, I'v created "Draft Pull Request label" and we can place on PR s if it is in draft mode and should net be reviewed yet. ofcource PR's with this label shouldn't be merged

@kartlos99 @shalva97 @nmgalo @mlkway @likaGlonti

**Comment by shalva97** [comment]
ხალხი ქართულად ვილაპარაკოთ, რათ გვინდა ინგლისური. Engrish bad, ქართული კაია!

**Reply by nmgalo** [reply]
![IMG_3647](https://user-images.githubusercontent.com/33604954/161030448-83d89fd2-6a87-4eac-ace9-37decf4911f9.JPG)

😂😂😂

**Reply by Nodrex** [reply]
> ხალხი ქართულად ვილაპარაკოთ, რათ გვინდა ინგლისური. Engrish bad, ქართული კაია!

Since we are going to open source project, we need to  use international language, cause our contributors may not be only Georgians
